pkg/route/api/v1beta3/conversion.go

```diff
@@ -9,6 +9,19 @@ func init() {
 		func(obj *RouteSpec) {
 			obj.To.Kind = "Service"
 		},
+		func(obj *TLSConfig) {
+			if len(obj.Termination) == 0 && len(obj.DestinationCACertificate) == 0 {
+				obj.Termination = TLSTerminationEdge
+			}
+			switch obj.Termination {
+			case TLSTerminationType("Reencrypt"):
```

Comment:
The constants are [lowercased](https://github.com/openshift/origin/blob/2092cbbaa9f5f26dc2973f6d14d7c0e474b0655e/pkg/route/api/types.go#L108-L113). This takes into account words starting with a capital letter (and defaults to the constants).